src/index.js

```diff
@@ -67,7 +67,6 @@ const processPullRequest = async ({ headMetrics, job, octokit, owner, prNumber,
 const run = async function () {
   const { baseBranch, commitSha, job, owner, prNumber, ref, repo, rootPath, title, token } = getInputs()
   const headMetrics = await readDeltaFiles(rootPath)
-  const isHeadBranch = ref === `refs/heads/${baseBranch}`
```

Comment:
I inlined this directly in the `if` further down, to save one statement because the maximum is 15 according to eslint.

Reply:
:old-man-yells-at-eslint:

if this is the result of that eslint rule, then we should disbale it.